lib/jexlTranformsMap.js

```diff
@@ -65,7 +65,9 @@ const map = {
         let s = new Set(arr);
         s.delete(x);
         return Array.from(s);
-    }
+    },
+    touppercase: (val) => String(val).toUpperCase(),
+    tolowercase: (val) => String(val).toLowerCase()
```

Comment:
As far as I remember, there is a .md document with a list of built-in functions. These two ones should be added to it.

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1210/commits/debc55d9426dcf4ab0d9c125dcc120ccab419253